README.md

```diff
@@ -65,7 +65,7 @@ If you need to stop the server from running, hit `Ctrl + C` and the server will
 In order to use a local copy of the CMR you will need to download the latest file, set an environment variable, and run a rake task to set required permissions and ingest some data.
 
 #### 1. Downloading the CMR file
-Go to https://ci.earthdata.nasa.gov/browse/CMR-CSB/latestSuccessful/artifact/, and download the `cmr-dev-system-uberjar.jar` file.
+Go to https://ci.earthdata.nasa.gov/browse/CN2-CSN2/latestSuccessful/artifact/, and download the `cmr-dev-system-uberjar.jar` file.
```

Comment:
The link was broken when cmr transitioned; this is just updating it to work again.

Reply:
I was able to download jar